### ISSUE TITLE
[Feature Request] Show count of similar cases in Alerts page

### ISSUE THREAD

**Issue by H2Cyber** [TheHive-Project/TheHive#2092]
### Request Type

Feature Request

### Feature Description

In the alerts page, show the number of similar cases for each alert :
![TheHive-Similar Cases](https://user-images.githubusercontent.com/764189/122942696-4dcbd780-d36e-11eb-85cf-c04406c16fc8.png)

### Rationale
This would help analysts to quickly find out if alerts are new or if they are related to previous cases.

**Comment by prot0sub** [comment]
Yes please, it would be awesome to have the number of similar cases displayed straight away in the Alerts pane !
